package.json

```diff
@@ -41,7 +41,7 @@
     "lint:styles": "stylelint \"{projects,feature-libs}/**/*.scss\"",
     "prettier": "prettier --config ./.prettierrc --list-different \"{projects,feature-libs,core-libs,integration-libs}/**/*{.ts,.js,.json,.scss,.html}\"",
     "prettier:fix": "prettier --config ./.prettierrc --list-different \"{projects,feature-libs,core-libs,integration-libs}/**/*{.ts,.js,.json,.scss,.html}\" --write",
-    "start": "env-cmd -e ci,b2c,$SPA_ENV ng serve",
+    "start": "env-cmd -e ci,b2b,$SPA_ENV ng serve",
```

Comment:
why did you change this?

Reply:
Sorry, that was by mistake. I was running b2b for another ticket. I will change